Tests/Functional/ParamFetcherTest.php

```diff
@@ -28,6 +28,23 @@ class ParamFetcherTest extends WebTestCase
         'foobar' => 'foo',
     ];
 
+    private function createUploadedFile($path, $originalName, $mimeType = null, $error = null, $test = false)
+    {
+        $ref = new \ReflectionClass('Symfony\Component\HttpFoundation\File\UploadedFile');
+        $params = $ref->getConstructor()->getParameters();
+
+        if ('error' === $params[3]->getName()) {
```

Comment:
This was not easy to spot at all. Introduced with https://github.com/symfony/symfony/pull/25471 and that was the reason of failing tests